tasks/marketplacesvm-push-disk-images/tests/mocks.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+set -ex
```

Comment:
Now that the main script was updated to `set -eux`, this one should probably use it as well.

Reply:
Patched 😄

Reply:
Thanks :)